File: sticktest/sticktest_test.go
package sticktest_test

import (
	"testing"

	"github.com/hsmtkk/fuzzy-giggle/sticktest"
	"github.com/stretchr/testify/assert"
)

func Test1(t *testing.T) {
	prob := sticktest.Problem{3, []bool{true, false, true, true, true}}
	want := true
	got := sticktest.Solve(prob)
	assert.Equal(t, want, got)
}

func Test2(t *testing.T) {
	prob := sticktest.Problem{3, []bool{true, false, false, false, false, true}}
	want := false
	got := sticktest.Solve(prob)
	assert.Equal(t, want, got)
}

File: sticktest/sticktest_test.py
import unittest

import sticktest


class TestStick(unittest.TestCase):
    def test1(self):
        p = sticktest.Problem(3, [True, False, True, True, True])
        want = True
        got = sticktest.solve(p)
        self.assertEqual(want, got)

    def test2(self):
        p = sticktest.Problem(3, [True, False, False, False, False, True])
        want = False
        got = sticktest.solve(p)
        self.assertEqual(want, got)
